tests/build/_internal/test_cmake_file.py

```diff
@@ -7,60 +7,51 @@
 from tests.build._internal.config.factories import ConfigFactory
 from mbed_tools.build._internal.cmake_file import generate_mbed_config_cmake_file, _render_mbed_config_cmake_template
 
+@pytest.fixture()
+def fake_target():
+    target = dict()
+    target["labels"] = ["foo"]
+    target["extra_labels"] = ["morefoo"]
+    target["features"] = ["bar"]
+    target["components"] = ["baz"]
+    target["macros"] = ["macbaz"]
+    target["device_has"] = ["stuff"]
+    target["c_lib"] = ["c_lib"]
+    target["core"] = ["core"]
+    target["printf_lib"] = ["printf_lib"]
+    target["supported_form_factors"] = ["arduino"]
+    target["supported_c_libs"] = {"loo": ["ginormous"]}
+    target["supported_application_profiles"] = ["full", "bare-metal"]
+    yield target
```

Comment:
We can just use `return` here as we don't need to pass control back to the fixture after the caller has finished executing in this case. We would use `yield` if we wanted the fixture to act as a context manager, or we needed to perform some tear-down step after the test has completed.